crow/index.md

```diff
@@ -58,37 +56,23 @@ Using the custom `[crow]` object in Max 8, create your own Live-controllable dev
 
 We have also created Max for Live devices to integrate crow with Ableton Live, including: synced clocks, MIDI-to-v/8, CC-to-voltage (which can be mapped to any software modulation source), i2c communication with [Just Friends](https://www.whimsicalraps.com/products/just-friends?variant=5586981781533), executing Lua code directly in Live, parameter mapping your crow scripts, and triggering Lua chunks with MIDI.
 
-Want to learn more? [Check out the docs.](/docs/crow/max-m4l/)  
+Want to learn more? [Check out the docs](/docs/crow/max-m4l/).  
 Afterward, visit the [Max and Max for Live repo on GitHub](https://github.com/monome/crow-max) to get started.
 
 ### scripting reference
 
-If you are writing or modifying norns apps, standalone scripts, or Max patches, you will want to become fluent in the language of the birds. Visit the [scripting reference](reference) to become a crow whisperer
-
-## Updates
-
-We are working all the time. Check out the [newest firmware version](https://github.com/monome/crow/releases/latest).
-
-To update, visit the step by step [bootloader instructions](update).
-
-## Help
-
-Answers to frequently asked questions can be found in [crow questions](faq).
-
-Community discussion happens at [llllllll.co](https://llllllll.co). Come say hello!
-
-Contact *help@monome.org* with further questions.
-
-## Calibration
-
-There are also a subset of commands for managing the state of the device and contents of flash memory. crow ships pre-calibrated, but it is possible to re-run the automatic calibration.
-See the [technical](technical) page for further details.
+If you are writing or modifying norns scripts, standalone druid snippets, or Max patches, you'll want to visit the [scripting reference](reference) to become fluent in the language of the birds.
 
+## Updates + Development
```

Comment:
updates and development are pretty disparate topics, so i'd split them up. updates probably doesn't need to mention the github link (it's out of context here).

Reply:
super good call, ty!